components/fl-orchestrator/pkg/proto/fl_orchestrator.proto

```diff
@@ -2,20 +2,50 @@ syntax = "proto3";
 
 option go_package = "github.com/katulu-io/fl-suite/fl-orchestrator-proto/service/fl_orchestrator";
 
-message ServersRequest {}
+message OperatorMessage {}
 
-message ServerResponse {
-  string RunID = 1;
-  string WorkflowName = 2;
-  string ClientImage = 3;
-  string ServerSNI = 4;
-}
+message OrchestratorMessage {
+  message ExecutorSpec {
+    message  OCIExecutorSpec {
+      string image = 1;
+    }
+
+    message  CoreMLExecutorSpec {
+      string model = 1;
+    }
+
+    message  TensorflowJSExecutorSpec {
+      string model = 1;
+    }
+
+    oneof executor {
+      OCIExecutorSpec oci_executor = 1;
+      CoreMLExecutorSpec coreml_executor = 2;
+      TensorflowJSExecutorSpec tensorflowjs_executor = 3;
+    }
+  }
+
+  message TaskSpec {
+    // Unique task identifier derived from the pipeline run and task id.
+    string ID = 1;
+    // Name of the workflow
+    string Workflow = 2;
+    // Server name indication used to identify the aggreation server.
+    string SNI = 3;
+
+    ExecutorSpec executor = 4;
+  }
+
+  message TasksSpec {
+    repeated TaskSpec tasks = 1;
+  }
 
-message GetServersResponse {
-  repeated ServerResponse Servers = 1;
+  oneof message {
+    TasksSpec tasks = 1;
+  }
```

Comment:
`repeated` fields are not allowed in `oneof`. IMO, let's remove the `oneof` in that case and have `repeated TaskSpec tasks = 1;` here instead. Otherwise one would have to use the API as `message.tasks.tasks`, let's make sure that we won't have the redundant `tasks` here and can use the API as `message.tasks`.